src/main/java/mods/eln/sixnode/lampsupply/LampSupplyElement.java

```diff
@@ -230,13 +230,12 @@ public String multiMeterString() {
     @NotNull
     @Override
     public Map<String, String> getWaila() {
-        Map<String, String> info = new HashMap<String, String>();
+        Map<String, String> info = new LinkedHashMap<String, String>();
```

Comment:
This isn't the correct data type. It's a Kotlin map. Shouldn't need this change?